### ISSUE TITLE
cannot see images from a binary plugin

### ISSUE THREAD

**Issue by thomasguo** [bertramdev/asset-pipeline#226]
We created a plugin called "rock".  This plugin includes some basic functionalities which could be reused by multiple projects.  It contains domain classes, controllers, gsp views, as well as many assets. And it uses asset-pipeline 2.1.1.   It works well if we run "grails run-app" command in the plugin project directory.  We packaged  it as a jar file with binary mode ('grails package-plugin --binary').  Then we used it in an application project.  All go well except all images from this plugin don't display in web pages. We checked the jar file and those images files are in this jar file(eg. /META-INF/assets/xxx.jpg ).  Anyone could help please?

More information about our case are like below:
---  JDK1.7, Grails 2.4.4 , asset-pipeline 2.1.1
---  In grails dev mode, only image files missed.  Javascript and css files are OK.
---  In grails prod mode(run as a war file),  all asset files missed.
---  If we package this plugin in normal zip format('grails package-plugin'), all work fine.

**Comment by davydotcom** [comment]
Very odd sorry for delayed response been moving. You have any more insights yet before I dig in?

**Comment by davydotcom** [comment]
Fixed in 2.1.3

**Comment by thomasguo** [comment]
Thanks. It works now.

**Comment by davydotcom** [comment]
No problem ! Sorry for the delay. 
--
David Estes

On February 26, 2015 at 3:17:51 AM, thomasguo (notifications@github.com) wrote:

Thanks. It works now.

**Comment by antonyperez** [comment]
Hi
I have a problem with my application, I have a core-app as a main project and I am creating plugins to integrate more models for it, also I am using Extjs for front-end, every plugin contains domain classes, controllers, services, views and assets, all of this is deploy to a local maven repository, 
Plugin-xx
grails-app
---controllers
---domain
---services
---views
---assets
------javascripts
---------plugin
------------controller
------------locale
------------model
------------store
------------view

When we use grails.plugin.location.' plugin-xx ' = "../plugin-xx " in the BuildConfig.groovy to add a plugin to the core-app  all works fine, but when I packaged a plugin as a jar file with binary mode  ( grails maven-deploy "--repository=XX" –binary) and we used it in the core-app  and execute “grails run-app” the core-app can´t access to the jar assets/locale files, we checked the jar file and those files are in /META-INF/assets/plugin/locale/plugin-lang-classes.js. 

The odd thing is that all Extjs code in the controller, model, store and view folders of the jar works fine.

Another thing is that when I execute “grails run-war” my jar
/META-INF/assets/plugin/
controller
	locale
	model
	store
	view
folders are not found.

I am using
- JDK1.7, Grails 2.3.7, asset-pipeline 2.1.4, Extjs 4.2.1
